frontends/common/options.cpp

```diff
@@ -89,6 +89,12 @@ CompilerOptions::CompilerOptions() : Util::Options(defaultMessage) {
     registerOption("--testJson", nullptr,
                     [this](const char*) { debugJson = true; return true; },
                     "Dump and undump the IR");
+    registerOption("--pi-file", "file",
+                   [this](const char* arg) { piFile = arg; return true; },
+                   "Write a PI control plane API description to the specified file.");
+    registerOption("--pi-as-json", nullptr,
+                   [this](const char* arg) { piAsJson = true; return true; },
+                   "Write out the PI API description as human-readable JSON.");
```

Comment:
I'm gonna have to push back against this one, Mihai. =) This JSON is explicitly a human-readable representation of the binary format we're using in the PI toolchain itself. JSON is not always easy to read, sure, but it's drastically easier than breaking out a hex editor. =)

It's true that the PI acronym is not well established now, but it will be very shortly once those tools are released to the broader community. I think the existing word is best. If you don't know what PI is, you can't really use the generated file in any case.